### PR TITLE
Hackathon/Codelab vision sample

### PR DESCRIPTION
**Pull request by ryanmats**
A work in progress for EDU hackathons / Codelab.

### CODE REVIEW

**Comment by ryanmats** [comment]
Still TODO: use Jinja2 for HTML output, README instructions

File: appengine/flexible/vision/main.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```
Comment (jonparrott) [inline appengine/flexible/vision/main.py]: 2017?

File: appengine/flexible/vision/main.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+import logging
+
+from flask import Flask, redirect, render_template, request
+
+from google.cloud import datastore
+from google.cloud import storage
+from google.cloud import vision
+
+
+CLOUD_STORAGE_BUCKET = '<your-storage-bucket>'
```
Comment (jonparrott) [inline appengine/flexible/vision/main.py]: Use os.environ and env_variables in app.yaml? (or do you have a good reason for this?)

**Comment by ryanmats** [comment]
will add test(s) soon

**Comment by jonparrott** [comment]
@ryanmats this looks good, I'd just prefer we put it under `/codelabs/flex_and_vision` or similar.

**Comment by jonparrott** [comment]
Please create a bug for yourself to add tests.

**Pull request merged**